diff --git a/node_modules/@ionic/angular/fesm2022/ionic-angular-common.mjs b/node_modules/@ionic/angular/fesm2022/ionic-angular-common.mjs
index 34ee45d..9c5f08d 100644
--- a/node_modules/@ionic/angular/fesm2022/ionic-angular-common.mjs
+++ b/node_modules/@ionic/angular/fesm2022/ionic-angular-common.mjs
@@ -4,7 +4,7 @@ import * as i3 from '@angular/router';
 import { NavigationStart, NavigationCancel, NavigationError, PRIMARY_OUTLET, ChildrenOutletContexts, ActivatedRoute, Router } from '@angular/router';
 import * as i1 from '@angular/common';
 import { DOCUMENT } from '@angular/common';
-import { isPlatform, getPlatforms, LIFECYCLE_WILL_ENTER, LIFECYCLE_DID_ENTER, LIFECYCLE_WILL_LEAVE, LIFECYCLE_DID_LEAVE, LIFECYCLE_WILL_UNLOAD, componentOnReady } from '@ionic/core/components';
+import { isPlatform, getPlatforms, LIFECYCLE_WILL_ENTER, LIFECYCLE_DID_ENTER, LIFECYCLE_WILL_LEAVE, LIFECYCLE_DID_LEAVE, LIFECYCLE_WILL_UNLOAD, componentOnReady } from '@ionic/core/components/index.js';
 import { Subject, fromEvent, BehaviorSubject, combineLatest, of } from 'rxjs';
 import { __decorate } from 'tslib';
 import { filter, switchMap, distinctUntilChanged } from 'rxjs/operators';
diff --git a/node_modules/@ionic/angular/fesm2022/ionic-angular-standalone.mjs b/node_modules/@ionic/angular/fesm2022/ionic-angular-standalone.mjs
index 5bacbf6..6aee4af 100644
--- a/node_modules/@ionic/angular/fesm2022/ionic-angular-standalone.mjs
+++ b/node_modules/@ionic/angular/fesm2022/ionic-angular-standalone.mjs
@@ -86,8 +86,8 @@ import { defineCustomElement as defineCustomElement$1b } from '@ionic/core/compo
 import { defineCustomElement as defineCustomElement$1c } from '@ionic/core/components/ion-title.js';
 import { defineCustomElement as defineCustomElement$1d } from '@ionic/core/components/ion-toast.js';
 import { defineCustomElement as defineCustomElement$1e } from '@ionic/core/components/ion-toolbar.js';
-import { modalController, popoverController, initialize, actionSheetController, alertController, createAnimation, getTimeGivenProgression, createGesture, loadingController, menuController, pickerController, toastController } from '@ionic/core/components';
-export { IonicSafeString, IonicSlides, createAnimation, createGesture, getPlatforms, getTimeGivenProgression, iosTransitionAnimation, isPlatform, mdTransitionAnimation } from '@ionic/core/components';
+import { modalController, popoverController, initialize, actionSheetController, alertController, createAnimation, getTimeGivenProgression, createGesture, loadingController, menuController, pickerController, toastController } from '@ionic/core/components/index.js';
+export { IonicSafeString, IonicSlides, createAnimation, createGesture, getPlatforms, getTimeGivenProgression, iosTransitionAnimation, isPlatform, mdTransitionAnimation } from '@ionic/core/components/index.js';
 import { defineCustomElement as defineCustomElement$1f } from '@ionic/core/components/ion-nav.js';
 import { NG_VALUE_ACCESSOR } from '@angular/forms';
 import { defineCustomElement as defineCustomElement$1g } from '@ionic/core/components/ion-checkbox.js';
